Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ruby:2.6.3
```

Comment:
Is there a way we can use whatever version we have in the `.ruby-version` file here? Just asking around

Reply:
mhhhmmm. To be honest, I don't know, I'll look into it!

Reply:
@juanmanuelramallo I did not find a way of doing this. If the ruby version is changed later, it will have to be changed here also. Is there a way to setup a reminder of sorts in github for this?

Reply:
I'm not sure about the reminder, it's fine as it is.